EIPS/eip-5143.md

```diff
@@ -0,0 +1,224 @@
+---
+eip: 5143
+title: Slippage Protection for Tokenized Vault
+description: An extension of the ERC-4626 standard for improved EOA interactions.
```

Comment:
We try to avoid standard in titles and descriptions:

```suggestion
description: An extension of ERC-4626 supporting improved EOA interactions.
```